### DISCUSSION TITLE
Ignore Code Block

### DISCUSSION THREAD

**Discussion by AdamGusstProject** [rojopolis/spellcheck-github-actions#209] [Ideas]
Would it be possible to build in functionality that ignores code blocks?  As we build out documentation, I noticed it's always the code blocks that are getting flagged for spelling issues.

I know there's an ignore function, but I've not figured out how to get that to work.

**Comment by AdamGusstProject** [comment]
Found a thread for this

**Reply by ccoVeille** [reply]
Can be you share it there then?

Thanks

**Reply by AdamGusstProject** [reply]
I just found this link talking about superfencing:

https://github.com/rojopolis/spellcheck-github-actions/discussions/99

**Comment by jonasbn** [comment]
Hi @AdamGusstProject 

How to accomplish this is already described in the documentation, do let me know if the documentation is not clear on this.

REF: https://github.com/rojopolis/spellcheck-github-actions?tab=readme-ov-file#specify-code-not-to-have-spelling-checked

@ccoVeille same request to you, if the documentation is not clear, please let me know.

**Reply by ccoVeille** [reply]
Thanks

My message was more about asking author to share what it found,because often a discussion can be found via Google, but here there would be no solution, as the author said somehow "I found it by myself"

I'm glad you are posting the exact link to the solution

Thanks

**Reply by jonasbn** [reply]
I would also like to see what @AdamGusstProject found, perhaps the documentation is unclear and requires improvements.

Feedback is always welcome.

**Reply by AdamGusstProject** [reply]
> Hi @AdamGusstProject
> 
> How to accomplish this is already described in the documentation, do let me know if the documentation is not clear on this.
> 
> REF: https://github.com/rojopolis/spellcheck-github-actions?tab=readme-ov-file#specify-code-not-to-have-spelling-checked
> 
> @ccoVeille same request to you, if the documentation is not clear, please let me know.

I have all this configured and it's still not ignoring code blocks.  Is there something more that needs to be added?

**Reply by facelessuser** [reply]
A minimal, reproducible example would be required for someone to begin offering advice. You could be doing something wrong, but without being able to see exactly what you've done, it is impossible to tell
